contrib/mod_tls.c

```diff
@@ -5787,7 +5850,7 @@ static int tls_ticket_key_cb(SSL *ssl, unsigned char *key_name,
 # endif /* OpenSSL-1.0.0 and later */
 
     memcpy(key_name, k->key_name, 16);
-    return 0;
+    return 1;
```

Comment:
This is the actual bug; we were returning the wrong value to OpenSSL to indicate a successful ticket generation.